custom_components/solarman/inverter_definitions/sofar_g3hyd.yaml

```diff
@@ -1,5 +1,8 @@
 # Sofar G3 also HYD 5-20KTL-3PH
-# some value commented after month no change
+# This works also for rebranded ZCS Azzurro 3-Phase inverters such as the 3PH HYD6000 ZSS
+# Note that this won't work if your ZCS inverter is connected via Connext, you have to be using a Wi-Fi or Ethernet Kit such as ZSM-WIFI-USB.
+# Defined but commented Battery 2 as that was empty during my tests.
```

Comment:
Do you have two batteries attached to the inverter? If not, that may explain why it's blank. Would be better IMO to surface the values, and let the user choose to ignore them.

Same for Module Temperature 2-3 and Radiator Temperature 2-6

Reply:
Sadly not, I only have one single battery pack (1 BMS + 9 Batteries from Pylontech).  Happy to remove the comment so that they get surfaced anyway and then users can decide to hide/ignore them.

Reply:
Yeah please surface that information.